src/utils/event.ts

```diff
@@ -88,3 +88,13 @@ export const eventTimeComparator = (a: VenueEvent, b: VenueEvent) => {
 
   return a.duration_minutes - b.duration_minutes;
 };
+
+export const getEventDayRange = (daysInBetween: number) => {
+  if (daysInBetween === 2) {
+    // add 1 day to daysInBetween to form a full timeline, if we already have exactly 2 days difference
+    return daysInBetween + 1;
+  }
+
+  // add 2 days to daysInBetween to form a full timeline
+  return daysInBetween + 2;
```

Comment:
why do we need 2 days here?.. shouldn't we always add just +1?

see here: the last date is disabled because it does not have events. 
![image](https://user-images.githubusercontent.com/63313732/128729500-9f1c7aff-886e-4823-b0e6-a52a1eabe4ea.png)

Reply:
fixed this logic in 60ff2eb
we still need 2 days, but only if there's 1 event and it's within current date, in that case we're adding 2 days
otherwise 1 day is enough to correctly display all dates